headers/headers.go

```diff
@@ -0,0 +1,75 @@
+// headers package provides header name constants and get/set functions for commonly used http headers in the
+// dp-api-clients-go repo. Package replaces go-ns lib and should be treated as the single source of truth
```

Comment:
Likely worth us updating the readme in go-ns to mention this too

Reply:
Its next on my list 😉